applications/frontend/views.py

```
@@ -0,0 +1,76 @@
+# -*- coding: utf-8 -*-
```

Ruff (0.14.5)

UTF-8 encoding declaration is unnecessary

Remove unnecessary coding comment

(UP009)